### ISSUE TITLE
Button for logging out other sessions

### ISSUE THREAD

**Issue by engelgabriel** [RocketChat/Rocket.Chat#1725]
Today, it's important to provide users with the ability to log out of not only their current session, but also any sessions they may have open from other devices. Otherwise what is a user to do if their mobile phone is lost or stolen, with a live login session still on the device? With Meteor Accounts this is as simple as providing a button that when clicked calls `Meteor.logoutOtherClients`

**Comment by alayek** [comment]
Hi @engelgabriel, I am interested in trying my hand in this issue; and get started with contributing to it.

I have basic understandings of Meteor framework, and I have built a few small applications in Meteor. 

I have already checked out Meteor forums regarding this `logoutOtherClients()` function. Kindly guide me on how I should go about raising the pull request.

I have already gone through https://github.com/RocketChat/Rocket.Chat/wiki/How-can-I-contribute-to-Rocket-Chat%3F

**Comment by marceloschmidt** [comment]
@alayek are you still interested in taking this? Are you familiar with Github, Forks and Pull Requests? I can guide you through, if you need assistance. Thank you for your interest.

**Comment by alayek** [comment]
@marceloschmidt yes! I am familiar with GitHub, Forks and Pull Requests. I have already forked this repo, and cloned locally. I will have to create a branch, and work on it.

Since this is a small change, I will have to rebase my commits to one single commit before I push my branch changes to GitHub. And then I can raise the PR from GitHub UI.

Did I miss something? Please add if that's the case. 

I would need some guidance regarding the technical aspect of it. I am good with JS, but new to Meteor framework. I need to know where the button would be placed in the UI, CSS classes to be used, and how to test my changes etc.

**Comment by marceloschmidt** [comment]
@alayek I'm not sure about the rebase part, as I would think it is possible to create a PR from your commits without rebasing, but if that works, good :) 

I think I'd add this button in the profile settings `packages/rocketchat-ui-account/account/accountProfile.html`, maybe below the password field? Or even at the bottom of everything else... 
CSS changes should go into `/packages/rocketchat-ui-account/styles/account.less`, although I think the current CSS is at `packages/rocketchat-theme/assets/stylesheets/base.less`. If you add anything to the first file (`account.less`), you need also to add this line `api.addAssets('styles/account.less', 'client');` in `/packages/rocketchat-ui-account/package.js` file. 

Once you create your PR, there's an automatic process that will deploy your changes to Heroku, where you can test them. Look for something like this `engelgabriel deployed to rocket-chat-pr-xxxx xx hours ago` on your PR info a few minutes after you create it. 

For testing your changes, you can access the Heroku generated URL on one browser, register and log in, then open another browser (or an anonymous window), log in and click the button, and check if the other browser has been logged out.

Let me know if you need any further help.

**Comment by alayek** [comment]
Sure thanks. I will try it out and let you know. :)

**Comment by alayek** [comment]
Hey @marceloschmidt, I have raised the pull-request with a simple implementation. I have tested locally with two browsers. I have also created more than one user accounts to test it - that logging out one user, doesn't affect other users' login sessions.

But I am still waiting for the Heroku URL.

I might need to modify the button CSS (placement of the button) and fix the hard-coded string on the button label. Kindly tell me how to do that.
